cnf-certification-test/accesscontrol/resources/resources.go

```diff
@@ -40,3 +40,38 @@ func HasRequestsAndLimitsSet(cut *provider.Container) bool {
 	}
 	return passed
 }
+
+// For more info on cpu mgmt polcies see https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/.
+func HasExclusiveCPUsAssigned(cut *provider.Container) bool {
```

Comment:
This seems to be equivalent to pre-exisitng function: IsPodGuaranteed() / GetGuaranteedPods()

Reply:
Yes, it's very similar but I need more granularity than Pod level because I need to determine if the containers inside a Pod have CPUs in both the exclusive and and the shared pool to fail the test. I understand that only using the shared CPU pool would be fine for this test.

With regards to the function you propose I think it has a misleading name because a Pod can belong to the QoS `Guaranteed` without having integer CPU units. See https://kubernetes.io/docs/tasks/administer-cluster/cpu-management-policies/. I have updated that function in https://github.com/test-network-function/cnf-certification-test/pull/790.

Also, reviewing the documentation I see that I need to check for memory limits as well in order for the Pod to qualify as `Guaranteed` Qos.